mods/ra/rules/vehicles.yaml

```diff
@@ -539,7 +539,10 @@ MRJ:
 	WithIdleOverlay@SPINNER:
 		Sequence: spinner
 		Offset: -256,0,256
+	GrantConditionOnDisabled:
```

Comment:
I may be forgetting something, but I don't think RA has anything that could trigger this for vehicles?  Should be able to skip this.